### PR TITLE
Devenv: Universal jwt_proxy environment

### PR DESCRIPTION
**Pull request by mgyongyosi**
**What this PR does / why we need it**:
This PR makes `jwt_proxy` work on MacOS, Linux and Windows. 

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

**Review by Jguer — APPROVED** [review]

File: devenv/docker/blocks/jwt_proxy/docker-compose.yaml [hunk 1]
```diff
@@ -33,22 +33,26 @@
...
+      "--redirect-url=http://env.grafana.local:8088/oauth2/callback",
+      "--pass-access-token=true",
+      "--email-domain=*",
     ]
-    network_mode: "host"
+    # network_mode: "host"
     depends_on:
       - oauthkeycloak
+    extra_hosts:
+      - "env.grafana.local:host-gateway"
```
Comment (eleijonmarck) [inline devenv/docker/blocks/jwt_proxy/docker-compose.yaml:55]: i have not seen this. does this work for all os? 🤔

Comment (Jguer) [reply devenv/docker/blocks/jwt_proxy/docker-compose.yaml:55]: tested on linux working here :+1: 

@mgyongyosi tested mac and windows

Comment (mgyongyosi) [reply devenv/docker/blocks/jwt_proxy/docker-compose.yaml:55]: It isn't documented extensively, but 'host-gateway' should resolve to the host's address on every platform.

**Review by linoman — APPROVED** [review]
I've tested this on MacOS (M2) and it does work as expected. Great job!

**Review by eleijonmarck — APPROVED** [review]

**Pull request merged**